### ISSUE TITLE
⌨ Unclear on what is selected or not on functional community details

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13343]
### 🛡️Fable tester: Michael S. 
When I went to the view community settings page, I was unable to determine by navigating the page with Voiceover whether the right selections were added to the community. 

This is because under the various options such as work streams or other roles, all of the options were displayed there, regardless of whether I had added them or not. 

For example, under work streams, options such as budget and resource management were displayed even though I know I did not add them to the community. There was no indication on this page of which selections I added and which I had not. 

The only way I was able to find out if the selections were correct is by clicking the edit this community link which brought up the form I had filled out before. 

The form was accessible so I could use it to determine if the right selections were aded. This could be very confusing for screen reader users and make it difficult for them to know which selections they made when adding the community.

### Expected behaviour 
Aria labels should be clear for what is selected vs not selected for forms. 

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

<img width="1430" alt="Image" src="https://github.com/user-attachments/assets/e33767e3-0b28-4764-a7bc-700972d1735f" />

### 🙋‍♀️ Proposed Solution

@GCTC-NTGC/designers Should we be showing false values on the functional communities?

**Comment by gobyrne** [comment]
This is similar to #12894 but in this case it is actually a fail. In this case, I think we can follow Maddy's advise and hide all the non-selected ones.

@JoshBeveridge are you ok with that intervention here?

Alternatively, (and maybe we should do it anyway) If we're going to use these icons in this way, they need to work with screen readers. Can we come up with an approach please?
